projects/RabbitMQ.Client/client/impl/ConsumerDispatching/ConsumerDispatcher.cs

```diff
@@ -26,26 +25,25 @@ protected override async Task ProcessChannelAsync()
                             try
                             {
                                 IBasicConsumer consumer = work.Consumer;
-                                string? consumerTag = work.ConsumerTag;
```

Comment:
Consider `string consumerTag = work.ConsumerTag!;`. With this approach you will reduce scope of change in this file.